### ISSUE TITLE
<AlanView /> can not be removed

### ISSUE THREAD

**Issue by Somnus007** [alan-ai/alan-sdk-reactnative#12]
In my case, I want to display the Alan button after the user logs in. It works well. But when the user logs out, the Alan button still exists, even though its parent component has been removed

**Comment by Somnus007** [comment]
Hi @snyuryev , any updates on this?

**Comment by snyuryev** [comment]
Hi @Somnus007 

Will be fixed with one of the nearest release. 
You can use "hide" method as workaround

**Comment by Somnus007** [comment]
Hi @snyuryev , it seems there is also an error with `hideButton` method.

![Screenshot_20211103-171112_EndwireApp](https://user-images.githubusercontent.com/31230649/140035108-4a70403f-c509-443a-8467-773c99d056ba.png)

**Comment by snyuryev** [comment]
@Somnus007 got it. Let us release new version with correct behaviour for removing view.
How do you remove view btw? Do you have separate app screen without Alan Button?

**Comment by Somnus007** [comment]
Hi @snyuryev , I tried to remove alan button in the most common way before.
`{show && <AlanView projectid='id' />}`

Now I'm trying `hideButton` method, but the method got error above.
```
handleToggleShowAlan = () => {
    const {show} = this.state;
    this.setState({show: !show}, () => {
      if (show) {
        this.AlanManager.hideButton();
      } else {
        this.AlanManager.showButton();
      }
    });
  };

<View>
  ...
  <AlanView projectid='id' />
</View>
```

**Comment by snyuryev** [comment]
Fixed with version 1.10.0
